src/test-run/commands/validations/argument.js

```diff
@@ -57,6 +58,15 @@ export function nonEmptyStringArgument (argument, val, createError) {
         throw createError('""');
 }
 
+export function nullableStringArgument (argument, val) {
+    if (val === null) return;
+
+    var type = typeof val;
+
+    if (type !== 'string')
```

Comment:
I more like in one line:
```javascript 
if (typeof val !== 'string' && val !== null)
```

Reply:
We use `type` as an argument of the `ActionNullableStringArgumentError` error. There is no reason to call `typeof` twice, so I'd leave this as is

Reply:
@churkin Are you agree with my point?

Reply:
@AlexanderMoskovkin  ok, but we can merge conditions:
```javascript
var type = typeof val;

if (type !== 'string' && val !== null)
    ...
```